docs/elasticsearch-specification.asciidoc

```diff
@@ -506,3 +507,26 @@ spec:
 include::orchestration.asciidoc[]
 include::advanced-node-scheduling.asciidoc[]
 include::snapshots.asciidoc[]
+
+
+[id="{p}-readiness"]
+=== Readiness probe
+
+By default, the readiness probe checks that the pod can successfully respond to HTTP requests within a three second timeout. This is acceptable in most cases. In some cases (such as when the cluster is under heavy load), it may be helpful to increase the timeout. This allows the pod to stay in a `Ready` state and thus be part of the Elasticsearch service even if it is responding slowly. To adjust the timeout, set the `READINESS_PROBE_TIMEOUT` environment variable in the pod template. For example, to increase it to ten seconds:
+
+[source,yaml]
+----
+spec:
+  nodeSets:
+  - name: default
+    count: 1
+    podTemplate:
+      spec:
+        containers:
+        - name: elasticsearch
+          env:
+          - name: READINESS_PROBE_TIMEOUT
+            value: 10
```

Comment:
I fear that this is not enough because the probe has also a timeout defined at the container level in the spec (set to 5sec).

https://github.com/elastic/cloud-on-k8s/blob/0728ae29afce0b79d9e04ef7d0d26a6597900d7c/pkg/controller/elasticsearch/nodespec/readiness_probe.go#L14-L20

Reply:
Good catch, I updated the docs with an example and tested it out